backend/config.py

```diff
@@ -310,6 +310,12 @@ def __getattr__(self, key):
         shutil.copyfile(frontend_favicon, STATIC_DIR / "favicon.png")
     except PermissionError:
         logging.error(f"No write permission to {STATIC_DIR / 'favicon.png'}")
+    except OSError as e:
```

Comment:
Let's just do Exception as e here to catch all errors.

Reply:
updated